IoTuring/Configurator/ConfiguratorIO.py

```diff
@@ -0,0 +1,144 @@
+import platform # Platform detection
+
+import json
+import os  # Configurations file path manipulation, environment variables
+import inspect # Configurations file path manipulation
+
+from IoTuring.Logger.LogObject import LogObject
+from IoTuring.MyApp.App import App # App name
+
+# macOS dep (in PyObjC)
+try:
+    from AppKit import *
+    from Foundation import *
+    macos_support = True
+except:
+    macos_support = False
+
+CONFIG_PATH_ENV_VAR = "IOTURING_CONFIG_DIR"
+
+CONFIGURATION_FILE_NAME = "configurations.json"
+
+# read ConfiguratorIO.checkConfigurationFileInOldLocation for more information
+DONT_MOVE_FILE_FILENAME = "dontmoveconf.itg"
+
+class ConfiguratorIO(LogObject):
+    def __init__(self):
+        self.directoryName = App.getName()
+    
+    def readConfigurations(self):
+        """ Returns configurations dictionary. If does not exist the file where it should be stored, return None. """
+        config = None
+        try:
+            with open(self.getFilePath(), "r") as f:
+                config = json.loads(f.read())
+            self.Log(self.LOG_MESSAGE, "Loaded \"" + self.getFilePath() + "\"")
+        except:
+            self.Log(self.LOG_WARNING, "It seems you haven't a configuration yet. Ensure you're using the configuration mode (-c) to enable your favourite entites and warehouses.\
```

Comment:
Grammar.

(Related issue, some way Logger should read the width of the terminal and crop long lines. This line looks strange because of this)

```suggestion
            self.Log(self.LOG_WARNING, "It seems you don't have a configuration yet. Use configuration mode (-c) to enable your favourite entities and warehouses.\
```